src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -37,6 +37,7 @@ object Backend {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+  type A <: BackendCall
```

Comment:
And of course `BackendCall` won't work because there already is a `BackendCall` that's a superclass of this abstract type. Thanks @mcovarr :)